jmbitcoin/jmbitcoin/bip21.py

```diff
@@ -44,10 +44,10 @@ def decode_bip21_uri(uri):
     return result
 
 
-def encode_bip21_uri(address, params):
+def encode_bip21_uri(address, params, safe=""):
```

Comment:
Is `safe` parameter here really needed? Can't we just always use ":/"?

Reply:
My reasoning was: this problem is specific to having a url inside the parameters of a url, so I thought that's a fairly specific case and maybe for other cases it isn't needed.